Misc/NEWS.d/next/C API/2021-12-11-08-41-36.bpo-45855.Lq2_gR.rst

```diff
@@ -0,0 +1 @@
+Replaced deprecated :c:func:`PyImport_ImportModuleNoBlock` with :c:func:`PyImport_ImportModule`. Patch by Kumar Aditya.
```

Comment:
Such a wording may create an impression that the whole Python (C API included) makes a replacement.

```suggestion
Built-in Python modules stopped using deprecated :c:func:`PyImport_ImportModuleNoBlock` in favor of :c:func:`PyImport_ImportModule`. Patch by Kumar Aditya.
```